apps/studio/src/components/sidebar/core/table_list/Item.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <stateless-sidebar-folder
+    v-if="source.type === 'schema'"
+    v-show="!source.hidden"
+    :expanded="source.expanded"
```

Comment:
It's not in this PR, but does this new system properly hook up with column lazy loading? Somewhere presumably we're dispatching `loadTableColumns` to the store?

Reply:
Oh, I don't think so. Let me check.